assets/js/components/ExecutionResults/ExecutionResults.jsx

```diff
@@ -111,46 +123,44 @@ function ExecutionResults({
         catalogError={false}
         clusterID={clusterID}
         hasAlreadyChecksResults
-        selectedChecks={checks}
+        selectedChecks={getChecks(checkResults)}
```

Comment:
I'm not sure if we really need to use the `getChecks` function here.
If you follow how this variable is used, it is simple used to check if it has any value on the list or not, so having I think sending `checkResults` or the `ids` of the checks would be the same